src/EventStore.Client.Streams/SystemStreams.cs

```diff
@@ -1,14 +1,51 @@
 #nullable enable
 namespace EventStore.Client {
+	/// <summary>
+	/// A collection of constants and methods to identify streams.
+	/// </summary>
 	public static class SystemStreams {
+		/// <summary>
+		/// A stream containing all streams in the EventStoreDB.
```

Comment:
We should clarify what this means as this sentence makes it seem like the events are duplicated in this stream. We are storing links in this stream.
```suggestion
		/// A stream containing links to events in the EventStoreDB.
```